.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      compiler: clang
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+        env:
+          - MATRIX_EVAL="export CC=clang && CXX=clang++"
```

Comment:
`export CC=clang CXX=clang++` ? Your statement is parsed as `(export CC=clang) && (CXX=clang++)` so the latter is not exported.

FYI, most of the travis CI with MATRIX_EVAL don't use `export`. https://docs.travis-ci.com/user/languages/c/  https://github.com/google/re2/blob/master/.travis.yml Maybe you can just throw `export` away.

Reply:
Done.